pom.xml

```diff
@@ -19,7 +19,7 @@
 
   <groupId>co.cask.wrangler</groupId>
   <artifactId>wrangler</artifactId>
-  <version>4.0.0-SNAPSHOT</version>
```

Comment:
We don't need to bump the version yet. since we have not released 4.0.0 version.

Reply:
fixed!